Enhances `spo theme list` (#5346)
pnp/cli-microsoft365: docs/docs/v7-upgrade-guidance.mdx

Update your scripts to use the list item id for `id` property instead of UniqueId product id.

> ## Enhanced `spo theme list`

Reviewer: We didn't really enhance the command. Let's use something like: `Updated spo theme list command output`.

> In version 7 of the CLI For Microsoft 365, we have made an enhancement for the data returned by the `spo theme list` command. Currently, when no items were found, we would only log an output when `--verbose` was specified. This made this command very unpredictable to use in scripts. This has now been changed so an empty array will be logged when no items were found. 

Reviewer: Stuff like "currently" and "current version" references this version is `v7`. When talking about that is actually now in our stable release, we should use "previously" or "previous version".

```suggestion
In version 7 of the CLI For Microsoft 365, we have made an adjustment for the data returned by the `spo theme list` command. Previously, when no items were found the command had no output. This made this command very unpredictable to use in scripts. The command has now been updated so an empty array will be logged when no items are found. 
```

### What action do I need to take?

> Make sure that your script expects an empty array instead of nothing when calling the command.

Reviewer: ```suggestion
Make sure that your script expects an empty array when there are no themes found.
```
